django_redis/hash_ring.py

```diff
@@ -1,19 +1,20 @@
 import bisect
 import hashlib
+from typing import Dict, Iterable, Iterator, List, Optional, Tuple
 
 
 class HashRing:
-    nodes = []
+    nodes = []  # type: List[str]
 
-    def __init__(self, nodes=(), replicas=128):
-        self.replicas = replicas
-        self.ring = {}
-        self.sorted_keys = []
+    def __init__(self, nodes: Iterable[str] = (), replicas: int = 128):
+        self.replicas: int = replicas
+        self.ring: Dict[str, str] = {}
+        self.sorted_keys: List[str] = []
 
         for node in nodes:
             self.add_node(node)
 
-    def add_node(self, node):
+    def add_node(self, node: str):
```

Comment:
I don't think this is automatically implied so you might as well add it to this and the others

```suggestion
    def add_node(self, node: str) -> None:
```

Reply:
Done